doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,828 @@
+
+There are Ansible [playbooks](https://galaxy.ansible.com/pbench/agent) available via Ansible Galaxy to install the `pbench-agent`, and the pieces needed (key and configuration files) to be able to send results to a server.  To use the RPMs provided above via COPR with the [playbooks](https://galaxy.ansible.com/pbench/agent), an inventory file needs to include the `fedoraproject_username` variable set to `portante`, for example:
+
+```
+...
+
+[servers:vars]
+fedoraproject_username: portante
+
+...
+```
+
+Alternatively, one can specify `fedoraproject_username` on the command line, rather than having it specified in the inventory file:
+
+    ansible-playbook -i <inventory> <playbook> -e '{fedoraproject_username: portante}'
+
+_**NOTE WELL**_: If the inventory file also has a definition for `pbench_repo_url_prefix` (which was standard practice before `fedoraproject_username` was introduced), it needs to be deleted, otherwise it will override the default repo URL and the `fedoraproject_username` change is not going to take effect.
+
+While we don't include installation instructions for the new `node-exporter` and `dcgm` tools in the published documentation, you can find a manual installation procedure for the Prometheus "node_exporter" and references to the Nvidia "DCGM" documentation in the [`agent/tool-scripts/README`](https://github.com/distributed-system-analysis/pbench/blob/v0.71.0-alpha.0/agent/tool-scripts/README.md).
+
+Container images built using the above RPMs are available in the [Pbench](https://quay.io/organization/pbench) organization in the Quay.io container image repository using tags `beta`, `v0.71.0-XX`, and `XXXXXXXXX`.
+
+
+Summary of Changes
+====
+
+## Support for RHEL 9 & CentOS 9
+
+Support for RHEL & CentOS 9 is provided in this release.  Note that since RHEL 9 has not been GA'd yet there might still be some changes that will have to be made to support it.
+
+
+## The New "Tool Meister" Sub-System
+
+The "Tool Meister" sub-system (introduced by PR #1248) is the major piece of functionality delivered with the release of `v0.71` of the pbench-agent.
+
+This is a significant change, where the pbench-agent first orchestrates the instantiation of a "Tool Meister" process on all hosts registered with tools, using a Redis instance to coordinate their operation, and the new "Tool Data Sink" process handles the collection of data into the pbench run directory hierarchy.  This effectively eliminates all remote SSH operations for individual tools except one per host to orchestrate the creation of the Tool Meister instance.
+
+One Tool Meister instance is created per registered host, and then a single Tool Data Sink instance is created on the host where the benchmark script is run.  The Tool Data Sink is responsible for collecting and storing locally all data sent to it from the deployed Tool Meister instances.
```

Comment:
That's an interesting point, because there are some circumlocutions here. But given that "controller" sat at the heart of identifying datasets on 0.69, and we're trying to clear ourselves of that baggage, I understand that trying to redefine it is awkward.

We could actually try "orchestrator" ... although it's a bit more complicated if the user is doing external orchestration.

But if we carefully and explicitly re-define "controller" to be the host where the benchmark wrapper script is run (and where we run TDS by default)... maybe ... ??

Reply:
So first of all, we don't have solid documentation on which to base the release notes, so there is not much here we can do.  Do we really want to introduce new abstractions to talk about the Pbench Agent sub-components being introduced here?  It seems much better to do that in the documentation.

For the purposes of the Release Notes the circumlocutions are probably the best we can do to avoid turning the Release Notes themselves into the new documentation.

Can we all agree to not solve this in the Release Notes?  Nobody reads them anyways. =)